<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Celsius 2 Fahrenheit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="6" columnstretch="1,0">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <item>
       <widget class="QGroupBox" name="celsiusBox">
        <property name="title">
         <string>摄氏</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="4,1">
         <item>
          <widget class="QDial" name="celsiusDial">
           <property name="minimum">
            <number>-20</number>
           </property>
           <property name="maximum">
            <number>150</number>
           </property>
           <property name="value">
            <number>26</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="celsiusLcd"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="fahrenheitBox">
        <property name="title">
         <string>华氏</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,1">
         <item>
          <widget class="QDial" name="fahrenheitDial">
           <property name="minimum">
            <number>-10</number>
           </property>
           <property name="maximum">
            <number>302</number>
           </property>
           <property name="value">
            <number>78</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="fahrenheitLcd"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
